lib/maze/server.rb

```diff
@@ -50,6 +50,7 @@ def reset_response_delay
       # @param type [String] Request type
       # @return Request list for the type given
       def list_for(type)
+        type = type.to_s if type.is_a? Symbol
         case type
```

Comment:
`to_s` is valid on a string (it just returns the string) so this can be simplified:

```suggestion
        case type.to_s
```